web/crux-ui/src/pages/products/[productId]/versions/[versionId]/deployments/[deploymentId].tsx

```diff
@@ -134,6 +134,11 @@ const DeploymentDetailsPage = (props: DeploymentDetailsPageProps) => {
       return
     }
 
+    if (deployment.node.status !== 'running') {
```

Comment:
I think we can check this at the beginning of the `onDeploy()` to reach the fail fast approach, what do you think?

Reply:
sure!